README.md

```diff
@@ -60,7 +60,7 @@ dependencies {
 
 How To (Compiling Jar From Source)
 ------
-To compile Paper, you need JDK 8, maven, and an internet connection.
+To compile Paper, you need JDK 16, maven, and an internet connection.
```

Comment:
I left this comment on the PR over on PaperDocs, we should specify Maven 3.6.3 or greater otherwise users will run into issues with maven on java 16.

Reply:
like say maven min version 3.6.3?

Reply:
It won't work either way as maven-shade-plugin breaks. See https://github.com/PaperMC/Paper/commit/5f9fd47688beec51a355866f949aede4e981e006#diff-aefb6d15d0b286bbeb5fc4daec5d392c5b7a317164c521620510e3e94dc67cfaR43-R44

Reply:
then i keep maven like always?

Reply:
Snapshot releases of maven-shade-plugin work on Java 16

Reply:
then any changes here?

Reply:
1.16 branch NEEDS java 8, not 16, and 16 has issues without other updates, so, while we recommend newer javas, afaik, can't compile with it on this branch nor is it a requirement

Reply:
Outdated because we won't be using Maven anymore. Keep on hold until 1.17 gets merged in.

Reply:
> 1.16 branch NEEDS java 8, not 16, and 16 has issues without other updates, so, while we recommend newer javas, afaik, can't compile with it on this branch nor is it a requirement

In teory 1.16 not require java 11? (when make this PR i put 11, when 1.17 release i change this again because master can be 1.17 in future)

Reply:
it requires java 8, 1.17 is where the java requirement was bumped to 11 by us, and then 16 by mojang; This section is only relevant when 1.17 is merged into master and this entire section needs re-doing anyways

Reply:
Okay. then i return to the cave because its cold...

Good luck :3